src/three/DebugTilesRenderer.js

```diff
@@ -372,11 +372,19 @@ export class DebugTilesRenderer extends TilesRenderer {
 
 		} else {
 
-			boxGroup.add( boxHelperGroup );
-			boxHelperGroup.updateMatrixWorld( true );
+			if ( boxHelperGroup ) {
 
-			sphereGroup.add( sphereHelper );
-			sphereHelper.updateMatrixWorld( true );
+				boxGroup.add( boxHelperGroup );
+				boxHelperGroup.updateMatrixWorld( true );
+
+			}
```

Comment:
We should also check above when removing the `boxHelperGroup` and `sphereHelper` to see if the objects are defined.

Reply:
Thanks for noticing. Done!

Reply:
Maybe something got lost in a rebase but I'm not seeing what you changed above -- I'm looking on lines 370 and 371 for a check to see if `boxHelperGroup` and `sphereHelper` got removed from the associated group

Reply:
ah, sorry! you're right. I added the checks to `disposeTile` but not to the if-branch of `setTileVisible`. Thanks for catching that! Should be good now.